<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tes3mp Server Browser</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout" name="centralLayout">
    <item>
     <layout class="QVBoxLayout" name="motdLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="motdWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>728</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>728</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabBrowser">
       <attribute name="title">
        <string>Browser</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="tblServerBrowser">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFavorites">
       <attribute name="title">
        <string>Favorites</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="tblFavorites">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxFilters">
      <property name="title">
       <string>Filters</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lblMaxLatency">
             <property name="text">
              <string>Max latency:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboLatency">
             <item>
              <property name="text">
               <string>All</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&lt;50</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&lt;100</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&lt;150</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&lt;200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&lt;250</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="lblGameMode">
             <property name="text">
              <string>Game mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leGamemode"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="cBoxNotFull">
           <property name="text">
            <string>Not full</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBoxWithPlayers">
           <property name="text">
            <string>With players</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cBBoxWOPass">
           <property name="text">
            <string>No password</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAccount"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QStatusBar" name="_statusBar"/>
  <action name="actionAdd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
